tests/labhub_test.py

```diff
@@ -106,10 +106,13 @@ def test_invite_cmd(self):
         mock_team_newcomers.is_member.return_value = False
         mock_team_developers.is_member.return_value = False
         mock_team_maintainers.is_member.return_value = False
+        self.mock_team.is_member.return_value = False
+        self.labhub.TEAMS = self.teams
 
-        testbot.assertCommand(
-            '!invite meet to newcomers',
-            'You need to be a member of this organization to use this command')
+        with self.assertRaises(queue.Empty):
+            testbot.assertCommand(
+                '!invite meet to newcomers',
+                ':poop:')
```

Comment:
something more crazy than `:poop:` ; that still feels like it might be a legitimate response from the command.